cmd/ocsp.go

```diff
@@ -0,0 +1,165 @@
+/*
+The MIT License (MIT)
```

Comment:
> Why is this file MIT licensed? If its from another package, can't it be vendorized?

Actually, we can vendorize but the code that I imported is short and simple (even the concept is so simple) so I thought it will be good if we avoid to import new dependencies and their dependencies.